docs/contributing/management.rst

```diff
@@ -31,7 +31,7 @@ Raising an issue
         or in any other public forum until we have had a chance to deal with it.
 
 Except in the case of security matters, of course, you're welcome to raise issues in any way that
-suits you - :ref:`on one of our email lists, or the IRC channel <community-resources>` or in person
+suits you - :ref:`on one of our email lists, or the IRC channel <development-community>` or in person
```

Comment:
What is this exactly? I'm not familiar with a IRC channel "development-community". Does this resolve into '#django-cms" somehow?